pkg/controller/service/service_controller.go

```diff
@@ -63,6 +62,14 @@ const (
 	notRetryable = false
 
 	doNotRetry = time.Duration(0)
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to kubeadm until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
I don't think this label should live here. Please move it back as a constant (duplicated until we have a "real" place for it) to `cmd/kubeadm/app/constants` _as well_. I don't want `cmd/kubeadm` to depend on `pkg/controller/service` just for this.

@dims OK for you? Can you please fix up?

Reply:
@luxas Done. Please see https://github.com/kubernetes/kubernetes/pull/53937